enterprise_subsidy/settings/base.py

```diff
@@ -364,3 +364,8 @@ def root(*path_fragments):
 
 # disable indexing on history_date
 SIMPLE_HISTORY_DATE_INDEX = False
+
+
+# How long we keep API Client data in cache. (seconds)
+ONE_HOUR = 60 * 60
+LMS_USER_DATA_CACHE_TIMEOUT = ONE_HOUR
```

Comment:
we may just want to cache this for much longer and rely on something else to invalidate + update